requirements.txt

```diff
@@ -12,6 +12,7 @@ fastapi[all]==0.82.0
 fideslang==1.2.0
 fideslib==3.1.2
 fideslog==1.2.3
+firebase-admin==5.3.0
```

Comment:
not sure if anything else is needed to bring in this dependency, but things seem to work with just this update locally